### PR TITLE
Adding Radial Control Sample

### PR DESCRIPTION
**Pull request by antoine-haas**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc-samples/9)

### CODE REVIEW

File: Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    self.radialController = [WUIRadialController createForCurrentView];
+
+    // Get the radial controller menu
+    WUIRadialControllerMenu* menu = self.radialController.menu;
+
+    // Get the menu items
+    NSMutableArray* menuItems = menu.items;
+
+    // Create a new menu item
+    WUIRadialControllerMenuItem* newMenuItem = [WUIRadialControllerMenuItem createFromKnownIcon:@"Custom Tool" value:WUIRadialControllerMenuKnownIconRuler];
```
Comment (bbowman) [inline Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m:139]: >WUIRadialControllerMenuKnownIconRuler [](start = 117, length = 37)

might add to the comment here about what this is / what the options are (custom icons)

File: Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+    // Add a new menu item
+    [menuItems addObject:newMenuItem];
+	
+    __block ViewController* blockSelf = self; // Ensures self will not be retained
```
Comment (bbowman) [inline Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m]: I'm not sure that comment is correct ...

**Comment by bbowman** [comment]
:clock1:

File: Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m (hunk 1 above)
Comment (antoine-haas) [reply Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m:139]: In the readme tutorial I give additional details about this pointing to the enumeration documentation. Do you think it's still worth to add more details in the code comments?

File: Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m (hunk 2 above)
Comment (antoine-haas) [reply Scenarios/Radial Control/RadialControlSample/RadialControlSample/ViewController.m]: Agreed, the comment is for __weak and not __block. I'm going to change the code to use __weak instead. Does that cover your concern?

File: Scenarios/Radial Control/README.md [hunk 3]
```diff
@@ -0,0 +1,308 @@
+# Using Projections: Radial Control
+
+Radial controllers like the Surface Dial allow Windows users to enable a host of compelling and unique user interaction experiences for Windows and Windows apps. They support a rotate action, a click action and a press and hold action. For more radial controllers terms and concepts, see [Surface Dial interactions](https://msdn.microsoft.com/en-us/windows/uwp/input-and-devices/windows-wheel-interactions).
```
Comment (andrejo-msft) [inline Scenarios/Radial Control/README.md]: For more **radial** controller terms and...

File: Scenarios/Radial Control/README.md [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+  [...]
+  
+  -(void)sliderDidChange{
+    self.sliderLabel.text = [NSString stringWithFormat:@"Slider value: %f", self.slider.value];
+  }
+
+  @end
+```
+
+You will now update this app to add a custom menu item to the radial controller tool that will respond to rotate and click actions by changing the slider value and toggling the swicth.
```
Comment (andrejo-msft) [inline Scenarios/Radial Control/README.md]: r/swicth/switch

**Review by andrejo-msft — COMMENTED** [review]
Small changes. This is a great feature, and I'm glad there's a sample.

**Pull request merged**